test/integration/domain/test_artifact.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+from clarity_ext.domain.artifact import StepInputOutputMapRepository
+from clarity_ext.clarity import ClaritySession
+import itertools
+
+
```

Comment:
Yes I agree, this is very much a hack as it's now. It would probably be easiest to maintain if the test would be set up beforehand by api calls.

But then again, it might be easier to maintain only the selenium tests when they have been set up.